src/Components/Facility/TriageForm.tsx

```diff
@@ -75,12 +75,17 @@ export const TriageForm = (props: triageFormProps) => {
           getTriageDetails({ facilityId: facilityId, id: id })
         );
         if (!status.aborted && res && res.data) {
+          // getting the entry date in new Date() format
+          const month = parseInt(res.data.entry_date.substring(5, 7)) - 1;
+          const date = parseInt(res.data.entry_date.substring(8, 10));
+          const year = parseInt(res.data.entry_date.substring(0, 4));
+          const entryDate = new Date();
+          entryDate.setFullYear(year, month, date);
+
```

Comment:
@Pranshu1902 why not use the formatDate function from utils?

Reply:
@khavinshankar how about now, I found a way to do it with `moment`